brd-ios/breadwallet/src/ViewModels/Transaction.swift

```diff
@@ -19,6 +19,13 @@ enum TransactionStatus {
     case complete
     /// Invalid / error
     case invalid
+    
+    /// Swap complete
+    case swapComplete = "COMPLETE"
+    /// Swap pending
+    case swapPending = "PENDING"
+    /// Swap failed
+    case swapFailed = "FAILED"
```

Comment:
do we need each case doubled? i mean.. whats the diff between cases "complete" and "swapComplete" could we not reuse the same cases?

Reply:
(prolly u only need a bool isSwap.. not 3 aditional status cases